### PR TITLE
Release/v0.32.9

### PR DESCRIPTION
**Pull request by marbar3778**
back port of: 

- [rpc/lib] [\#4248](https://github.com/tendermint/tendermint/issues/4248) RPC client basic authentication support (@greg-szabo)

- [metrics] \#4263 Add
  - `consensus_validator_power`: track your validators power
  - `consensus_validator_last_signed_height`: track at which height the validator last signed
  - `consensus_validator_missed_blocks`: total amount of missed blocks for a validator
    as gauges in prometheus for validator specific metrics

- [rpc/lib] [\#4051](https://github.com/tendermint/tendermint/pull/4131) Fix RPC client, which was previously resolving https protocol to http (@yenkhoon)
- [cs] \#4069 Don't panic when block meta is not found in store (@gregzaitsev)

* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

**Comment by marbar3778** [comment]
https://github.com/tendermint/tendermint/pull/4294#discussion_r364768118 there is this comment that we can add @melekes thoughts?

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -17,6 +17,17 @@ program](https://hackerone.com/tendermint).
 
 ### FEATURES:
 
+- [rpc/lib] [\#4248](https://github.com/tendermint/tendermint/issues/4248) RPC client basic authentication support (@greg-szabo)
```
Comment (melekes) [inline CHANGELOG_PENDING.md]: move to `changelog.md`?

Comment (melekes) [reply CHANGELOG_PENDING.md]: also don't see version update

Comment (marbar3778) [reply CHANGELOG_PENDING.md]: https://github.com/tendermint/tendermint/pull/4298, sorry a bit brain dead right now

**Comment by melekes** [comment]
> #4294 (comment) there is this comment that we can add @melekes thoughts?

should be easy to do. but it does not have to be in this release imo

**Pull request merged**